app/code/community/Bolt/Boltpay/Model/ShippingAndTax.php

```diff
@@ -195,10 +205,12 @@ public function getShippingAndTaxEstimate( Mage_Sales_Model_Quote $quote, $boltO
                 return $response;
             }
 
-            $this->applyShippingRate($quote, null);
+            $this->applyShippingRate($quote, null, false);
```

Comment:
@danac-gs 
When resetting the quote to no shipping rate applied, it is not necessary to collect shipping rates on the address.